addons/point_of_sale/models/pos_order.py

```diff
@@ -148,7 +148,12 @@ def _process_order(self, order, draft, existing_order):
         pos_order = pos_order.with_company(pos_order.company_id)
         self = self.with_company(pos_order.company_id)
         self._process_payment_lines(order, pos_order, pos_session, draft)
+        return pos_order._process_saved_order(draft)
 
+    @api.model
```

Comment:
```suggestion
```

remove this. @api.model is analogous to @staticmethod. meaning, self should just be the model, not the records.

Reply:
Thank you, I didn't know about that.